app/assets/javascripts/quby/answers/validation.js

```diff
@@ -126,7 +126,7 @@
                   if(value === undefined || value == ""){
                       continue;
                   }
-                  var rgx = /(\s*-?[1-9]+[0-9]*\s*|\s*-?[0-9]?\s*)/;
+                  var rgx = /(\s*-?[1-9]+[0-9]+\s*|\s*-?[0-9]\s*)/;
```

Comment:
Ik vind het geval dat 09 een typo is nogal waarschijnlijker dan dat het een bedoelde 09 is, vandaar dat ik geen leidende nullen wil toestaan voor integer velden.